holopy/scattering/theory/multisphere.py

```diff
@@ -39,6 +40,8 @@
                       MultisphereFailure)
 from .scatteringtheory import FortranTheory
 
+def normalize_polarization(illum_polarization):
```

Comment:
Is there any chance of this being used somewhere else? It seems like it could be useful to have it more accessible than inside multisphere...